client/lib/domains/assembler.js

```diff
@@ -37,8 +37,7 @@ function createDomainObjects( dataTransferObject ) {
 			privateDomain: domain.private_domain,
 			pendingTransfer: domain.pending_transfer,
 			registrar: domain.registrar,
-			registrationDate: domain.registration_date,
```

Comment:
Maybe we should store all dates, and then use them in the decorator to turn them into `moment` objects?